templates/blog/article.html

```diff
@@ -32,111 +34,167 @@
-          <li class="p-inline-list__item">
-            Share on:
-          </li>
+      {% if tags %}
+        <div class="p-block">
+          {%- for tag in tags -%}
+            <a class="p-chip--information" href="/blog/tag/{{ tag.slug }}">
+              <span class="p-chip__value">
+                  {{ tag.name }}
+              </span>
+            </a>
+          {%- endfor -%}
+        </div>
+      {% endif %}
 
+      <ul class="p-inline-list--social">
+        <li class="p-inline-list__item">
+          <span class="p-text--default">
+            Share on:
+          </span>
           <li class="p-inline-list__item">
-            <a  class="p-link--inverted p-icon--facebook" title="Share on Facebook" href="https://www.facebook.com/sharer/sharer.php?u=https://www.canonical.com/blog/{{ article.slug }}">
+            <a class="p-icon--facebook" title="Share on Facebook" href="https://www.facebook.com/sharer/sharer.php?u=https://www.canonical.com/blog/{{ article.slug }}">
               Facebook
             </a>
           </li>
 
           <li class="p-inline-list__item">
-            <a class="p-link--inverted p-icon--twitter" title="Share on Twitter" href="https://twitter.com/share?text={{ article.title.rendered|safe|urlencode }}&amp;url=https://www.canonical.com/blog/{{ article.slug }}&amp;hashtags=ubuntu">
+            <a class="p-icon--twitter" title="Share on Twitter" href="https://twitter.com/share?text={{ article.title.rendered|safe|urlencode }}&amp;url=https://www.canonical.com/blog/{{ article.slug }}&amp;hashtags=ubuntu">
               Twitter
             </a>
           </li>
 
           <li class="p-inline-list__item">
-            <a class="p-link--inverted p-icon--linkedin" title="Share on LinkedIn" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://www.canonical.com/blog/{{ article.slug }}&amp;title={{ article.title.rendered|safe|urlencode }}">
-            LinkedIn
-          </a>
+            <a class="p-icon--linkedin" title="Share on LinkedIn" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://www.canonical.com/blog/{{ article.slug }}&amp;title={{ article.title.rendered|safe|urlencode }}">
+              LinkedIn
+            </a>
           </li>
-        </ul>
-      </div>
+      </ul>
     </div>
-    {% if tags %}
-      <div class="row">
-        <div class="col-10">
-          <p>
-            <strong>Tags:</strong>
-            {%- for tag in tags -%}
-              <a href="/blog/tag/{{ tag.slug }}">
-                {{ tag.name }}
-              </a>{%- if not loop.last %},{% endif -%}
-            {%- endfor -%}
-          </p>
-        </div>
-      </div>
-    {% endif %}
   </header>
-
-  <section class="p-strip is-bordered">
-    <div class="row u-equal-height">
-      <div class="col-8">
-        <div>
-          {{ article.content.rendered|safe }}
+  <hr class="p-rule is-fixed-width">
+  <div class="p-article__row">
```

Comment:
```suggestion
  <div class="p-article__row">
    <hr class="p-rule">
```
Since it's just a row